app/components/post_draft/draft_input/draft_input.test.tsx

```diff
@@ -0,0 +1,166 @@
+        });
+
+        it('opens scheduled post options on long press', async () => {
+            // make this a re-usable function
+            await operator.handleConfigs({
+                configs: [
+                    {id: 'ScheduledPosts', value: 'true'},
+                ],
+                configsToDelete: [],
+                prepareRecordsOnly: false,
+            });
+
+            const {getByTestId} = renderWithEverything(<DraftInput {...baseProps}/>, {database});
+            fireEvent(getByTestId('draft_input.send_action.send.button'), 'longPress');
+            expect(openAsBottomSheet).toHaveBeenCalledWith(expect.objectContaining({
+                screen: Screens.SCHEDULED_POST_OPTIONS,
+            }));
+        });
+
+        it('handles persistent notifications', async () => {
+            jest.mocked(persistentNotificationsConfirmation).mockResolvedValueOnce();
+            const props = {
+                ...baseProps,
+                postPriority: {
+                    persistent_notifications: true,
+                    priority: PostPriorityType.URGENT,
+                } as PostPriority,
+                value: '@user1 @user2 message',
+            };
+
+            const {getByTestId} = renderWithEverything(<DraftInput {...props}/>, {database});
+            fireEvent.press(getByTestId('draft_input.send_action.send.button'));
+            expect(persistentNotificationsConfirmation).toHaveBeenCalled();
+        });
+    });
+
+    describe('Input Handling', () => {
+        it('updates text value', () => {
+            const {getByTestId} = renderWithEverything(<DraftInput {...baseProps}/>, {database});
+            fireEvent.changeText(getByTestId('draft_input.post.input'), 'new message');
+            expect(baseProps.updateValue).toHaveBeenCalledWith('new message');
+        });
+
+        it('handles cursor position', () => {
+            const {getByTestId} = renderWithEverything(<DraftInput {...baseProps}/>, {database});
+            fireEvent(getByTestId('draft_input.post.input'), 'selectionChange', {
+                nativeEvent: {selection: {start: 5, end: 5}},
+            });
+            expect(baseProps.updateCursorPosition).toHaveBeenCalledWith(5);
+        });
+
+        it('updates focus state', () => {
+            const {getByTestId} = renderWithEverything(<DraftInput {...baseProps}/>, {database});
+            fireEvent(getByTestId('draft_input.post.input'), 'focus');
+            expect(baseProps.setIsFocused).toHaveBeenCalledWith(true);
+        });
+    });
+
+    describe('Validation', () => {
+        it('disables send when canSend is false', () => {
+            const props = {...baseProps, canSend: false};
+            const {getByTestId} = renderWithEverything(<DraftInput {...props}/>, {database});
+            const sendButton = getByTestId('draft_input.send_action.send.button.disabled');
+            expect(sendButton).toBeTruthy();
+            expect(sendButton).toBeDisabled();
```

Comment:
Again, just as sanity check, we could double check that both press and long press actions do not call the functions, since the button is disabled.

But that may be already covered by the button being marked as disabled...

Reply:
Done.